### PR TITLE
week6: update journal

### PR DESCRIPTION
**Pull request by garrettbslone**

### CODE REVIEW

File: journals/week6.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	if (flux_aux_set (h, FRIPP_AUX_TAG, ctx, fripp_ctx_destroy) ==  -1) {
+		return  -1;
+	}
+}
+  
+if (inc)
+	return  fripp_format_send (ctx, "%s:+%ld|g", name, value);
+else
+	return  fripp_format_send (ctx, "%s:%ld|g", name, value);
+}
```
Comment (grondo) [inline journals/week6.md]: minor: the formatting of this code seems to be a little off.

**Review by grondo — APPROVED** [review]
Nice work last week!

Just a note to check on the formatting (indentation) of one of the code snippets. Otherwise, this can be merged and we can discuss how to move forward this week.

**Comment by garrettbslone** [comment]
Force pushed to fix formatting issue.

**Comment by grondo** [comment]
Great! Merging.

**Pull request merged**